terraform/environments/performance/main.tf

```diff
@@ -0,0 +1,115 @@
+locals {
+  # Azure Credits
+  arm_subscription_id = "330ffa29-65cb-4bfd-9004-4a3b3c8647de"
+
+  # Generate these by following
```

Comment:
Can we have this in a README in this directory perhaps?

Reply:
Updated